### PR TITLE
Use upstream NFD chart

### PR DESCRIPTION
**Pull request by e0ne**
NFD charts are merged to NFD repo now so we don't need to duplicate
effort of maintaining charts.

Closes: #68
Closes: #111

depends-on: kubernetes-ci#104

Signed-off-by: Ivan Kolodiazhnyi <ikolodiazhny@nvidia.com>

### CODE REVIEW

**Review by moshe010 — COMMENTED** [review]
Why we still need the folder  under deployment/network-operator. I thought all the point that we will have one chart in nfd isn't it?

**Comment by e0ne** [comment]
CI passed. I'll remove WIP once https://github.com/Mellanox/kubernetes-ci/pull/104 will be merged to not break other PRs

**Comment by e0ne** [comment]
Need to add minimum helm version

File: Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+apiVersion: v2
+name: network-operator
+version: 0.4.0
+kubeVersion: '>= 1.17.0'
+appVersion: v0.4.0
+description: Nvidia Mellanox network operator
```
Comment (moshe010) [inline Chart.yaml:6]: you can remove the Mellanox

File: Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+apiVersion: v2
```
Comment (moshe010) [inline Chart.yaml:1]: why we need a new Chart.yaml. Don't we have this already in https://github.com/Mellanox/network-operator/blob/master/deployment/network-operator/Chart.yaml

Comment (e0ne) [reply Chart.yaml:1]: It was a part of WIP patch. I'll remove it

**Review by moshe010 — APPROVED** [review]

**Pull request merged**